### PR TITLE
[CSP-7539] input schema docs

### PR DESCRIPTION
**Pull request by Kalininator**
Added documentation for writing input schemas. 

Haven't covered DDLs and `lookup` as those will be covered as a broader topic in a later ticket

### CODE REVIEW

File: docs/pages/input_schema/common_properties.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+```js
+{
+    public_url: {
+        type: 'string',
+        required: true,
+        defaultJsonPath: '$.env.public_url',
+    },
+}
+```
+
```
Comment (EdinaBolkeny) [inline docs/pages/input_schema/common_properties.md:157]: Question: should we mention`oneOf` as well?

File: docs/pages/input_schema/input_types.md [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+
+An input must have a specified type, or multiple types.
+The following input types are supported in the Tray properties panel.
+
+1. TOC
+{:toc}
+
+## String
+
+String fields require the user the enter some value as text.
```
Comment (EdinaBolkeny) [inline docs/pages/input_schema/input_types.md:21]: String fields require the user `to` enter ...

File: docs/pages/input_schema/input_types.md [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+    script: {
+        type: 'string',
+        format: 'code',
+    }
+}
+```
+
+## Number
+
+Number fields require the user the enter some value as a number.
```
Comment (EdinaBolkeny) [inline docs/pages/input_schema/input_types.md:95]: Number fields require the user `to` enter ...

File: docs/pages/input_schema/input_types.md [hunk 4]
```diff
@@ -0,0 +1,252 @@
...
+If the user entered `51` into the box, the connector would receive the following input.
+
+```json
+{
+    "house_number": 51
+}
+```
+
+Notice how this time you get a number, not a string.  
+If this was a `type: 'string'` field instead, the connector would have received a string instead.
```
Comment (EdinaBolkeny) [inline docs/pages/input_schema/input_types.md:117]: Do we need both insteads?

**Review by EdinaBolkeny — CHANGES_REQUESTED** [review]
Just a question regards oneOf and a couple of comments.

File: docs/pages/input_schema/common_properties.md (hunk 1 above)
Comment (Kalininator) [reply docs/pages/input_schema/common_properties.md:157]: https://trayio.atlassian.net/browse/CSP-7568
i will soon, its a bit of a bigger topic so separated it out

File: docs/pages/input_schema/input_types.md (hunk 4 above)
Comment (Kalininator) [reply docs/pages/input_schema/input_types.md]: good catches, thanks

**Review by EdinaBolkeny — APPROVED** [review]

File: docs/pages/input_schema/common_properties.md (hunk 1 above)
Comment (EdinaBolkeny) [reply docs/pages/input_schema/common_properties.md:157]: It makes sense.

**Pull request merged**